### PR TITLE
Add timeout counters for the "search" clients

### PR DESCRIPTION
**Pull request by dazoakley**
I missed these when doing the registers the other day... There are no (unit) tests on these clients, I'm happy to add some, but not really sure where to start - anyone for pairing?

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -31,6 +31,3 @@ generic-service:
 
 generic-prometheus-alerts:
   alertSeverity: ppud-replacement-alerts
-  extraDashboardTags:
```
Comment (AndyNewton) [inline helm_deploy/values-dev.yaml:34]: Unrelated clean-up?

**Review by AndyNewton — CHANGES_REQUESTED** [review]
I see no tests for the added funtionality @dazoakley .  Can we have some please!  Thanks.

File: helm_deploy/values-dev.yaml (hunk 1 above)
Comment (dazoakley) [reply helm_deploy/values-dev.yaml:34]: Yes, seemed pointless having a separate PR for it...

**Comment by dazoakley** [comment]
> I see no tests for the added funtionality @dazoakley . Can we have some please! Thanks.

So is that an offer to pair/help writing them then?

**Review by AndyNewton — APPROVED** [review]
